napalm_logs/listener.py

```diff
@@ -31,15 +31,15 @@ def start(self):
         '''
         Listen to messages and queue them.
         '''
+        buffer_size = 1024
```

Comment:
Same, perhaps we should move this as a global and renamed to `BUFFER_SIZE` to be more obvious.